### DISCUSSION TITLE
This repository is temporarily unavailable.

### DISCUSSION THREAD

**Discussion by sajidshahriar72543** [community/community#51585] [Repositories] [answered]
### Select Topic Area

Bug

### Body

I just made a repository to public from private.

Now it is looping on this page. Please help me.
![image](https://user-images.githubusercontent.com/63386478/229159525-46a769bb-1f8f-4212-af75-b7af5c635536.png)

**Comment by ramazancetinkaya** [comment, marked as answer]
@sajidshahriar72543, please kindly raise a ticket to [GitHub Support](https://support.github.com/) for the support engineers to investigate the issue.

They usually respond pretty quickly (within a couple of working days).

Hope this clarifies.

If you find this reply useful, please kindly 'Mark as answer' so that others know this issue is already addressed.

Have a good day.
